src/python/pose_format/utils/pose_converter.py

```diff
@@ -38,8 +38,147 @@
     {("pose_keypoints_2d", "RHeel"), ("POSE_LANDMARKS", "RIGHT_HEEL")},
 ]
 
-POSES_MAP = BODY_MAP + LEFT_HAND_MAP + RIGHT_HAND_MAP
+FACE_MAP = [
+    # face border
+    (("face_keypoints_2d", "FB_0"), ("FACE_LANDMARKS", "127")),
+    (("face_keypoints_2d", "FB_1"), ("FACE_LANDMARKS", "234")),
+    (("face_keypoints_2d", "FB_2"), ("FACE_LANDMARKS", "93")),
+    (("face_keypoints_2d", "FB_3"), ("FACE_LANDMARKS", "132")),
+    (("face_keypoints_2d", "FB_4"), ("FACE_LANDMARKS", "58")),
+    (("face_keypoints_2d", "FB_5"), ("FACE_LANDMARKS", "172")),
+    (("face_keypoints_2d", "FB_6"), ("FACE_LANDMARKS", "136")),
+    (("face_keypoints_2d", "FB_7"), ("FACE_LANDMARKS", "149")),
+    (("face_keypoints_2d", "FB_8"), ("FACE_LANDMARKS", "152")),
+    (("face_keypoints_2d", "FB_9"), ("FACE_LANDMARKS", "378")),
+    (("face_keypoints_2d", "FB_10"), ("FACE_LANDMARKS", "365")),
+    (("face_keypoints_2d", "FB_11"), ("FACE_LANDMARKS", "397")),
+    (("face_keypoints_2d", "FB_12"), ("FACE_LANDMARKS", "288")),
+    (("face_keypoints_2d", "FB_13"), ("FACE_LANDMARKS", "361")),
+    (("face_keypoints_2d", "FB_14"), ("FACE_LANDMARKS", "323")),
+    (("face_keypoints_2d", "FB_15"), ("FACE_LANDMARKS", "454")),
+    (("face_keypoints_2d", "FB_16"), ("FACE_LANDMARKS", "356")),
 
+    ("interpolate", {"avg_between": (("FACE_LANDMARKS", "136"), ("FACE_LANDMARKS", "149")), "avg_for": ("FACE_LANDMARKS", "150")}),
```

Comment:
I don't fully understnad this feature here - usually, when wanting to create an average, you should do (like done for the body)
```py
{("pose_keypoints_2d", "MidHip"), ("POSE_LANDMARKS", ("RIGHT_HIP", "LEFT_HIP"))},
```
Where `MidHip` in OpenPose is calculated from `RIGHT_HIP` and `LEFT_HIP`. 
So what it seems to me you are trying to do here is:
```py
{("face_keypoints_2d", ("FB_6", "FB_7"), ("POSE_LANDMARKS", "150")},
```

Which is the reverse - calculate a single MediaPipe point from two OpenPose points. Then, I believe none of the logic has to change, only the addition of `FACE_MAP`